environment.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - pip
   - tqdm
   - conda-forge::enlighten
-  - protobuf
```

Comment:
failure in dockernized unittest because of unspecified protobuf version:
<img width="842" alt="image" src="https://github.com/eth-easl/modyn/assets/100585941/c73cb0a8-c82b-439f-a243-1a14e04a4411">

Reply:
Wouldn't it be better to pin the protobuf version rather than removing the dependency completely?

Reply:
I did intend to pin a version but made a mistake... definitely we can't remove protobuf

Reply:
I revert this change because of the dedicated PR https://github.com/eth-easl/modyn/pull/412